client/signup/steps/design-type-with-store/siteground-store/siteground-logo.jsx

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
I've added `/** @ssr-ready **/` to the top. I'm not sure if we should be putting this in `SocialLogo` at this point, since from what I've figured, this is just a temporary test in order to gather data, and we might not need this logo after it is completed.